largest continuous subset/kadane.cpp

```diff
@@ -0,0 +1,56 @@
+// PROBLEM :: find the sum of the largest sum continuous array using KADANE'S ALGORITHM
+
+
+#include <iostream>
+#include <vector>
+using namespace std;
+
+
+void maxSumSubArray(vector<int>values)
+{
+	int max_so_far=values[0];
+	int curr_max=values[0];
+
+	int s=0,e=0;
+
+	int n=values.size();
+
+	for(int i=1;i<n;i++)
```

Comment:
You do not need to take an extra variable `n` to store the size of `values`.
You can instead do:
```C++
    for (int i = 1; i < values.size(); i++) {
        ... 
```
This wouldn't be a performance issue, and would improve code readability too. You may read [this StackOverflow answer](http://stackoverflow.com/a/3901666/5245129) for more on this topic.